docs/docsource/operation/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst

```diff
@@ -36,12 +36,46 @@ Click the **Storage Locations** item in the sidebar to open the
       * Scality RING with S3 Connector
       * Scality RING with sproxyd Connector
       * Ceph RADOS Gateway
-      * A Zenko local filesystem.
+      * A Zenko local filesystem
 
 #. Each storage location type has its own requirements. No security is
    required for a local file system, but all public clouds require
    authentication information.
 
+.. note::
+
+   Adding a location requires credentials (an access key and a secret key). 
+   Though nothing prevents you from using account-level credentials when
+   Zenko requests credentials for a location, it is a best practice to enter
+   a user ID specifically for this access. In other words, before you add a
```

Comment:
~~a user ID~~ `a user's credentials`
I think this conveys it more clearly that credentials are required not a user id